tests/symbolics/test_sympy_backend.py

```diff
@@ -178,3 +183,54 @@ def test_min_max_works_for_symbols(backend):
     values = symbols("a, b, c")
     assert backend.min(*values) == sympy_min(*values)
     assert backend.max(*values) == sympy_max(*values)
+
+
+def test_log2_expression():
```

Comment:
[issue] Why are we testing private functions here?
In general private functions should not be tested, because they should be an implementation detail. So if we decide to change them in future, we'll be forced to change tests as well. Tests should only. check the behaviour that's observable by the user and private functions by definition aren't really observable.